ts/graphs/TableData.svelte

```diff
@@ -29,6 +30,10 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
         text-align: end;
     }
 
+    .space {
+        padding-left: 5px;
+    }
+
```

Comment:
Defining an extra column for spacing feels like a bit of a hack - I imagine this could probably be accomplished by just styling the table or td tags instead?